change/@fluentui-react-tabs-79da62f4-21a0-4398-a9dd-e850571f6a38.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28773&commitId=8ea053e5066df292ea5d07324029726cf21e6ba4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance.normal.chromium.png|116354| Changed|
|Card Converged.CardHeader.normal.chromium.png|0| Removed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Removed|
|Card Converged.appearance - RTL.normal.chromium.png|0| Removed|
|Card Converged.card templates.normal.chromium.png|0| Removed|
|Card Converged.orientation.chromium.png|0| Removed|
|Card Converged.size.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|15162| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|11907| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|11907| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|14308| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|11790| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|15739| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>